src/pages/SalesLeadDeliveryPage/SalesLeadDeliveryPage.tsx

```diff
@@ -462,9 +503,46 @@ const ConfirmSection: React.FC<{
   const [numDeliver, setNumDeliver] = useState(1)
   const { properties } = useProperty()
   const [visible, setVisible] = useState(false)
+  const { currentUserRole } = useAuth()
+  const { id: appId, settings, enabledModules } = useApp()
+  const propertyColumn = JSON.parse(settings['sales_lead_delivery_page.confirm_section.default_member_property_column'])
+  const propertyColumnIds = propertyColumn
+    .map((columnName: string) => properties.find(property => columnName === property.name))
+    .map((property: Property) => property.id)
+  const [visibleColumnIds, setVisibleColumnIds] = useState<string[]>([
+    '#',
+    'createdAt',
+    'consumption',
+    'categories',
+    'managerName',
+    ...propertyColumnIds,
+  ])
+  const [fieldFilter, setFieldFilter] = useState<FiledFilter>({})
```

Comment:
It seems this state doesn't been used, so would it be possible to filter the data?

Reply:
This is used for MemberCollectionTableBlock

Reply:
Because the member data in lines +706 to +722 did'nt use the fieldFilter state, and the  soure of the MemberCollectionTableBlock's member data likely from this code,so I think it is not being used.

Reply:
If the MemberCollectionTableBlock use fieldFilter, does the MemberCollectionTableBlock still fetch the member data inside?

Reply:
I understand what you mean. Thank you for your review. I confirmed with the design that the search function is not needed and have removed it.